### ISSUE TITLE
Hi, what about paste spell improve?

### ISSUE THREAD

**Issue by Darkool** [nisovin/MagicSpells#11]
Maybe "paste" spell should check if player can build in this area, i mean, now players can paste schematics and overwrite another terrains, even worldguard zones, etc

**Comment by WolfieWaffle** [comment]
I totally agree. I would really like this, have it check if the area you would paste into only has and and replaceable blocks (like grass).
